### PR TITLE
[File System Access article] Add info about `startIn`, `suggestedName`, and `id`

### PR DESCRIPTION
**Pull request by tomayac**
Changes proposed in this pull request:

- Add info about `startIn`, `suggestedName`, and `id`.

### CODE REVIEW

File: src/site/content/en/blog/file-system-access/index.md [hunk 1]
```diff
@@ -234,6 +234,75 @@ file at a specific position, or resize the file.
   calling `close()` or when the stream is automatically closed by the pipe.
 {% endAside %}
 
+### Specifying a suggested file name and start directory
+
+In many cases your application may already have an opinion when it comes to default file names.
```
Comment (jpmedley) [inline src/site/content/en/blog/file-system-access/index.md:239]:
```suggestion
In many cases you may want your app to suggest a default file name or location.
```

Comment (jpmedley) [reply src/site/content/en/blog/file-system-access/index.md:239]: Two reasons:

* Opening sentence should cover both.
* Anthropomorphizing computers is a tech writing anti-pattern.

File: src/site/content/en/blog/file-system-access/index.md [hunk 2]
```diff
@@ -234,6 +234,75 @@ file at a specific position, or resize the file.
...
+  types: [{
+    description: 'Text documents',
+    accept: {
+      'text/plain': ['.txt'],
+    },
+  }],
+});
+```
+
+The same goes for the default start directory. A text editor may want to start the file save
```
Comment (jpmedley) [inline src/site/content/en/blog/file-system-access/index.md:256]:
```suggestion
The same goes for the default start directory. If you're building a text editor may want to start the file save
```

File: src/site/content/en/blog/file-system-access/index.md [hunk 3]
```diff
@@ -234,6 +234,75 @@ file at a specific position, or resize the file.
...
+or file open dialog in the default `documents` folder, whereas an image editor may want to
```
Comment (jpmedley) [inline src/site/content/en/blog/file-system-access/index.md:257]:
```suggestion
or file open dialog in the default `documents` folder, whereas for an image editor you may want to
```

File: src/site/content/en/blog/file-system-access/index.md [hunk 4]
```diff
@@ -234,6 +234,75 @@ file at a specific position, or resize the file.
...
+```
+
+The list of the well-known system directories is:
+
+- `desktop`: The user's desktop directory, if such a thing exists.
+- `documents`: Directory in which documents created by the user would typically be stored.
+- `downloads`: Directory where downloaded files would typically be stored.
+- `music`: Directory where audio files would typically be stored.
+- `pictures`: Directory where photos and other still images would typically be stored.
+- `videos`: Directory where videos/movies would typically be stored.
```
Comment (jpmedley) [inline src/site/content/en/blog/file-system-access/index.md:275]: I haven't used Windows in years. How platform independent is this list?

File: src/site/content/en/blog/file-system-access/index.md [hunk 5]
```diff
@@ -234,6 +234,75 @@ file at a specific position, or resize the file.
...
+  startIn: directoryHandle
+});
+```
+
+### Specifying the purpose of different file pickers
+
+Sometimes applications have different pickers for different purposes.
+For example, a rich text editor may allow the user to open text files,
+but also to import images. By default, each file picker would open at the last
+remembered location of the previous picker. You can circumvent this by storing `id`
```
Comment (jpmedley) [inline src/site/content/en/blog/file-system-access/index.md:292]: >at the last remembered location of the previous picker

This looks redundant. Am I missing something?

File: src/site/content/en/blog/file-system-access/index.md [hunk 6]
```diff
@@ -234,6 +234,75 @@ file at a specific position, or resize the file.
...
+values for each type of picker. If an `id` is specified, the file picker implementation
+will remember a separate last used directory for pickers with that same `id`.
```
Comment (jpmedley) [inline src/site/content/en/blog/file-system-access/index.md:294]:
```suggestion
will remember a separate last-used directory for pickers with that same `id`.
```

**Review by jpmedley — CHANGES_REQUESTED** [review]

File: src/site/content/en/blog/file-system-access/index.md (hunk 3 above)
Comment (tomayac) [inline src/site/content/en/blog/file-system-access/index.md:257]:
```suggestion
or file open dialog in the default `documents` folder, whereas for an image editor, may want to
```

File: src/site/content/en/blog/file-system-access/index.md (hunk 4 above)
Comment (tomayac) [reply src/site/content/en/blog/file-system-access/index.md:275]: Independent enough to work on CrOS, macOS, Windows, and Linux (I think the distribution was Ubuntu).

File: src/site/content/en/blog/file-system-access/index.md (hunk 5 above)
Comment (tomayac) [inline src/site/content/en/blog/file-system-access/index.md:292]:
```suggestion
remembered location. You can circumvent this by storing `id`
```

File: src/site/content/en/blog/file-system-access/index.md [hunk 7]
```diff
@@ -234,6 +234,75 @@ file at a specific position, or resize the file.
...
+const fileHandle = await self.showOpenFilePicker({
+  startIn: directoryHandle
+});
+```
+
+### Specifying the purpose of different file pickers
+
+Sometimes applications have different pickers for different purposes.
+For example, a rich text editor may allow the user to open text files,
+but also to import images. By default, each file picker would open at the last
```
Comment (tomayac) [inline src/site/content/en/blog/file-system-access/index.md:291]:
```suggestion
but also to import images. By default, each file picker would open at the last-remembered 
```

**Review by jpmedley — APPROVED** [review]

**Pull request merged**